src/Whim/Keybind/KeybindManager.cs

```diff
@@ -127,4 +127,13 @@ public bool Remove(string commandId)
 		_keybindsCommandsMap[keybind].Remove(commandId);
 		return true;
 	}
+
+	public void RemoveAll()
+	{
+		Logger.Debug("Removing all keybinds");
+		foreach (string commandId in _commandsKeybindsMap.Keys.ToArray())
+		{
+			Remove(commandId);
+		}
```

Comment:
I this should work?

```suggestion
		_commandsKeybindsMap.Clear();
```

Reply:
Maybe... but then what do we do about the equivalent of `_keybindsCommandsMap[keybind].Remove(commandId);` from `Remove()`?

Reply:
That's true - can you clear both maps?